### ISSUE TITLE
Support for Header Exchange Trigger

### ISSUE THREAD

**Issue by ianrathbone** [Azure/azure-functions-rabbitmq-extension#69]
Hello, thanks for adding official support for RabbitMQ.

Are there any plans to bring in support for Headers Exchange as a trigger instead of using a queue binding?

**Comment by ianrathbone** [comment]
I've now given this a go with the Pull Request: #71

**Comment by fabiocav** [comment]
FYI @jeffhollan

**Comment by jeffhollan** [comment]
Yep this comes up often.  Appears to be a different "flavor" of messages from RabbitMQ.  Would love to see this added as well as the feature for "durable" queues which is another issue

**Comment by cachai2** [comment]
Needs further investigation

**Comment by ianrathbone** [comment]
Hi, sorry I've not revisited this in some time - would it be preferable to close now given the time that has passed?
